flow-typed/npm/tone_v14.x.x.js

```diff
@@ -0,0 +1,82 @@
+declare module "tone" {
+    // TODO: There are two things called "Frequency" in Tone.js
+    //       https://tonejs.github.io/docs/14.7.58/type/Frequency
+    //       https://tonejs.github.io/docs/14.7.58/fn/Frequency
+    //       This "Frequency" is the type one, maybe call it FrequencyType?
+    declare type FrequencyType = string;
+
+    declare export class FrequencyClass {
+        toMidi: any
+    }
+
+    declare export function Frequency(frequency: number | string): FrequencyClass;
```

Comment:
Add comment: `// https://tonejs.github.io/docs/14.7.58/fn/Frequency`

Reply:
I think it's worth keeping our Flow types as close to the TypeScript types as we can. We can change the `Frequency` function type to:

    declare export function Frequency(value: TimeValue | FrequencyType ): FrequencyClass;

We have already created the `TimeValue` and `FrequencyType` types and can use those directly to match the TypeScript definition.

Reply:
Done.